README.md

```diff
@@ -22,12 +22,13 @@ CometBFT is a Byzantine Fault Tolerant (BFT) middleware that takes a
 state transition machine - written in any programming language - and securely
 replicates it on many machines.
 
-It is a fork of [Tendermint Core][tm-core] and implements the Tendermint
-consensus algorithm.
+CometBFT is the canonical implementation of the Tendermint consensus algorithm and is a
+primary building block for the [Interchain Stack](https://interchain.io/). Historically, 
+CometBFT has its roots and is a fork of [Tendermint Core][tm-core].
```

Comment:
I would avoid saying it is a fork as the project diverged so much from it. We could maybe say that historically project originated from tendermint core, but it evolved significantly from the freezed version of tendermint core.

Reply:
How do we politely say that project, Tendermint Core, was frozen and we are the continuity of it?

We probably have a blog post or something explaining this transition that we could link here, for anyone looking for more details.

Reply:
good ideas!

eb4450f66